dotCMS/src/test/java/com/dotmarketing/business/cache/provider/h22/H22CacheTest.java

```diff
@@ -268,4 +278,105 @@ public void run() {
 
 	}
 
+	public H22Cache newCacheInstance(){
+		File dir = Files.createTempDir();
+		dir.mkdirs();
+
+		final H22Cache cache = new H22Cache(dir.getAbsolutePath());
+		try {
+			cache.init();
+		} catch(Exception e) {
```

Comment:
Codacy found an issue: [Avoid catching generic exceptions such as NullPointerException, RuntimeException, Exception in try-catch block](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6190817)